packages/p2p/src/stores/buy-sell-store.js

```diff
@@ -1,3 +1,4 @@
+import React from 'react';
```

Comment:
why is react imported here?

Reply:
We need to import it since we are using jsx in the file. Please see `setLocalCurrencies` function.